logging-manager.js

```diff
@@ -241,8 +259,20 @@ const Logger = (module.exports = {
       if (this.checkInterval) {
         clearInterval(this.checkInterval)
       }
+      if (this.logLevelSource === 'file') {
+        this.getTracingEndTime = this.getTracingEndTimeFile
+      } else if (this.logLevelSource === 'gce_metadata') {
+        this.getTracingEndTime = this.getTracingEndTimeMetadata
+      } else if (this.logLevelSource === 'none') {
+        return
+      } else {
+        console.log('Unrecognised log level source')
+        return
+      }
       // check for log level override on startup
-      this.checkLogLevel()
+      this.checkLogLevel().catch((error) => {
+        console.log(error)
```

Comment:
`checkLogLevel` does not throw in the current revision, so we can remove the `catch` block.